docs/data-sources/certificate.md

```diff
@@ -0,0 +1,58 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "vcf_certificate Data Source - terraform-provider-vcf"
+subcategory: ""
+description: |-
+  Datasource used to extract certificate details for various resources based on fields like domain, issued_by, issued_to, key_size, and others.
+---
+
+# vcf_certificate (Data Source)
+
+Datasource used to extract certificate details for various resources based on fields like domain, issued_by, issued_to, key_size, and others.
```

Comment:
This message seems misleading, only `domain_id` and `resource_fqdn` are accepted as inputs

Reply:
Updated the MD file in the latest commit.

Reply:
Can you use the ` symbols to format?

Reply:
@tenthirtyam, completed the update as per your suggestion. Thank you.